corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -61,7 +61,9 @@ <h2 tabindex="0"><%- title %></h2>
     <div class="input-group">
       <input id="<%- text ? text : "" %>"
              type="text"
+             pattern="[0-9-/]+"
              class="date query-field form-control"
+             placeholder="<%- dateFormat %>"
```

Comment:
This parallels what I recently did in the form entry widget: https://github.com/dimagi/commcare-hq/pull/32433
A later commit in this PR unifies the two implementations